vnext/Microsoft.ReactNative/Views/FrameworkElementTransferProperties.cpp

```diff
@@ -45,8 +46,18 @@ void TransferFrameworkElementProperties(const xaml::DependencyObject &oldView, c
   TransferProperty(oldView, newView, xaml::FrameworkElement::MaxHeightProperty());
   TransferProperty(oldView, newView, xaml::FrameworkElement::FlowDirectionProperty());
   TransferProperty(oldView, newView, xaml::Controls::Canvas::ZIndexProperty());
-  TransferProperty(oldView, newView, winrt::Microsoft::ReactNative::ViewPanel::LeftProperty());
-  TransferProperty(oldView, newView, winrt::Microsoft::ReactNative::ViewPanel::TopProperty());
+
```

Comment:
Can we put this into a `TransferPropertyWithInvalidateArrange` function, or add a flag to the `TransfterProperty` function?